src/llmcompressor/modifiers/awq/base.py

```diff
@@ -590,43 +589,44 @@ def _forward_input_with_kwargs(
         """
         kwargs = input_kwargs or self._module_kwargs
         kwargs = _sanitize_kwargs(kwargs, module)
-        return tensor_forward_with_input_args(
-            module=module,
-            inputs=inputs,
-            input_kwargs=kwargs,
-        )[0]
+
+        inputs = inputs.to(next(module.parameters()).device)
```

Comment:
This will fail for offloaded modules (will return the meta device)
Instead, use `get_execution_device`. This kind of device movement can also be handled by the `IntermediatesCache`

Reply:
ok, updated. This is a vestige from AutoAWQ that we'll have to clean up with the pipelining refactor. We are doing `.to(device=...)` in several places in this file, which won't work for offloaded modules (ticket INFERENG-531)